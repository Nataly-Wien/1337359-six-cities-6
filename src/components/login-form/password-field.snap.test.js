import React from 'react';
import {render} from '@testing-library/react';
import PasswordField from './password-field';

it(`Should password input render correctly`, () => {
  const onChange = () => { };
  const password = `12345678`;
  const {container} = render(<PasswordField onChange={onChange} password={password} />);

  expect(container).toMatchSnapshot();
});
