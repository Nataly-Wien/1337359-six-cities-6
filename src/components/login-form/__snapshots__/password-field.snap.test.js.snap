// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should password input render correctly 1`] = `
<div>
  <input
    class="login__input form__input"
    name="password"
    placeholder="Password"
    type="password"
    value="12345678"
  />
</div>
`;
